sources/packages/backend/package.json

```diff
@@ -23,7 +23,7 @@
     "test:watch": "jest --watch ",
     "test:cov": "cross-env ENVIRONMENT=test jest --coverage --forceExit",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e:api": "npm run db:seed:test filter=CreateInstitutionsAndAuthenticationUsers,CreateAESTUsers,CreateStudentUsers && cross-env NODE_OPTIONS=--max_old_space_size=8192 ENVIRONMENT=test jest --collect-coverage --verbose --config ./apps/api/test/jest-e2e.json --forceExit",
+    "test:e2e:api": "npm run db:seed:test filter=CreateInstitutionsAndAuthenticationUsers,CreateAESTUsers,CreateStudentUsers && cross-env NODE_OPTIONS=--max_old_space_size=12288 ENVIRONMENT=test jest --verbose --config ./apps/api/test/jest-e2e.json --forceExit",
```

Comment:
Can you please explain, why --collect-coverage has been removed?

Reply:
@guru-aot `--collect-coverage` was removed as a test and the `bump` is explained in the PR description.

Reply:
Yes, it was a random bump that apparently did not help.